nomt/src/options.rs

```diff
@@ -47,4 +50,12 @@ impl Options {
     pub fn metrics(&mut self, metrics: bool) {
         self.metrics = metrics;
     }
+
+    /// Set the number of io_uring instances.
+    ///
+    /// Must be more then 1
```

Comment:
than.

Reply:
Why should it be more than one and not zero?

Reply:
because otherwise there will be input output at all

Reply:
sorry, I don't understand it. Can you rephrase?

Reply:
sorry, you're right, it must be more than *zero*